examples/broadcast/main.go

```diff
@@ -0,0 +1,172 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/pion/rtcp"
+	"github.com/pion/webrtc-v3-design"
+	"github.com/pion/webrtc/v3"
+
+	"github.com/pion/webrtc/v3/examples/internal/signal"
+)
+
+const (
+	rtcpPLIInterval = time.Second * 3
+)
+
+func main() {
+	sdpChan := signal.HTTPSDPServer()
+
+	// Everything below is the Pion WebRTC API, thanks for using it ❤️.
+	offer := webrtc.SessionDescription{}
+	signal.Decode(<-sdpChan, &offer)
+	fmt.Println("")
+
+	// Since we are answering use PayloadTypes declared by offerer
+	mediaEngine := webrtc.MediaEngine{}
+	err := mediaEngine.PopulateFromSDP(offer)
+	if err != nil {
+		panic(err)
+	}
+
+	// Create the API object with the MediaEngine
+	api := webrtc.NewAPI(webrtc.WithMediaEngine(mediaEngine))
+
+	peerConnectionConfig := webrtc.Configuration{
+		ICEServers: []webrtc.ICEServer{
+			{
+				URLs: []string{"stun:stun.l.google.com:19302"},
+			},
+		},
+	}
+
+	// Create a new RTCPeerConnection
+	peerConnection, err := api.NewPeerConnection(peerConnectionConfig)
+	if err != nil {
+		panic(err)
+	}
+
+	// Allow us to receive 1 video track
+	//FIXED-TO-V3 TODO
+	if _, err = peerConnection.AddTransceiverFromKind(webrtc.RTPCodecTypeVideo); err != nil {
+		panic(err)
+	}
+
+	//FIXED-TO-V3 Track=>LocalRTPTrack
+	localTrackChan := make(chan *webrtc.LocalRTPTrack)
+	// Set a handler for when a new remote track starts, this just distributes all our packets
+	// to connected peers
+	//FIXED-TO-V3 Track=>webrtc.RemoteRTPTrack
+	peerConnection.OnTrack(func(remoteTrack *webrtc.RemoteRTPTrack, receiver *webrtc.RTPReceiver) {
+		// Send a PLI on an interval so that the publisher is pushing a keyframe every rtcpPLIInterval
+		// This can be less wasteful by processing incoming RTCP events, then we would emit a NACK/PLI when a viewer requests it
+		go func() {
+			ticker := time.NewTicker(rtcpPLIInterval)
+			for range ticker.C {
+				if rtcpSendErr := peerConnection.WriteRTCP([]rtcp.Packet{&rtcp.PictureLossIndication{MediaSSRC: remoteTrack.SSRC()}}); rtcpSendErr != nil {
```

Comment:
I guess this should be
```suggestion
				if rtcpSendErr := receiver.WriteRTCP([]rtcp.Packet{&rtcp.PictureLossIndication{MediaSSRC: remoteTrack.SSRC()}}); rtcpSendErr != nil {
```